internal/worker/caasmodeloperator/modeloperator.go

```diff
@@ -114,6 +115,14 @@ func (m *ModelOperatorManager) update() error {
 			password = prevConf.OldPassword()
 			setPassword = false
 		}
+
+		// retrieves model operator deployment image to keep model operator's image the same after migration
+		modelImage, err := m.broker.GetModelOperatorDeploymentImage()
+		if err != nil {
+			return errors.Annotate(err, "failed to get model deployment image")
+		}
+
+		info.ImageDetails.RegistryPath = modelImage
```

Comment:
We don't want to replace the entire image path, just the registry bit.

Reply:
refactored w thanks